src/main/scala/org/holmesprocessing/totem/services/peinfo/service.conf.example

```diff
@@ -1,2 +1,5 @@
-[settings]
-Port        = 8080
+{
+	"settings":	{
+		"port" : 8080
```

Comment:
other services appear to all be using 
```
-	"HTTPBinding": ":8080", 
```
and no longer port after you recent pull request. Please adjust this to match